packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -2327,18 +2433,25 @@ def _job_params_label_key(ts):
         oracle_bucket = self.oracle_data_bucket
 
         # Register cloud storage on CVAT to pass user dataset
-        cvat_cloud_storage = cvat_api.create_cloudstorage(
-            **_make_cvat_cloud_storage_params(oracle_bucket)
+        _params = _make_cvat_cloud_storage_params(oracle_bucket)
+        _params["bucket_host"] = "http://minio:9010"
+        cvat_cloud_storage = cvat_api.create_cloudstorage(**_params)
+
+        segment_size = (
+            self.manifest.annotation.job_size or Config.cvat_config.cvat_task_segment_size
```

Comment:
Why there is `or`? Also, for skeleton tasks, there is a size multiplier currently (`DEFAULT_ASSIGNMENT_SIZE_MULTIPLIER`).

Reply:
Oh, I missed that `AnnotationInfo.job_size` is not nullable. I used `Config.cvat_config.cvat_task_segment_size` as backup value because it was [used](https://github.dev/humanprotocol/human-protocol/blob/dfefc91b976abcd35fa8a7f84582e7acc118ab49/packages/examples/cvat/exchange-oracle/src/cvat/api_calls.py#L307) previously when creating a task on CVAT. I'll fix.

Reply:
Just to clarify, in that case, we do not need to have the ability to configure task segment size based on some value from the exchange oracle config, right?

Reply:
I remember HP wanted the oracle to handle all of this, but there are arguments against it, so we haven't come to a final solution. Let's extract a function for this now.